pkg/apic/auth/apicauth.go

```diff
@@ -379,6 +381,7 @@ func (ptp *platformTokenGetter) Close() error {
 
 // fetchNewToken fetches a new token from the platform and updates the token cache.
 func (ptp *platformTokenGetter) fetchNewToken() (string, error) {
+	log.Debug("Get cached token is empty.  Try and fetch a new token")
```

Comment:
Trace or Debug?

Reply:
I was hemming and hawing... I'll move it to trace... don't have an issue with that